src/utils/window-console.js

```diff
@@ -129,6 +156,7 @@ export function logFriendlyPreamble() {
       %cwindow.dispatch%c - The function to dispatch a Redux action to change the state.
       %cwindow.actions%c - All the actions that can be dispatched to change the state.
       %cwindow.experimental%c - The object that holds flags of all the experimental features.
+      %cwindow.enablePseudoLocalization%c - Enable pseudo localizations by passing "accented" or "bidi" to this function.
```

Comment:
I thought that because it's mentioned when enabling I wouldn't need to add here.

TBH I wasn't completely happy about this word "enable", because this function also disables it. Maybe "set" instead?

Reply:
Yeah, I also thought about the same thing but wasn't completely satisfied with the alternatives I found 😄
I guess `set` would be better. Then maybe we can add another keyword, like "disabled", to disable this instead of an `undefined` value?

Reply:
`toggle` this will be :-)